packages/x-tailwindcss/demo/src/css/modal.css

```diff
@@ -0,0 +1,28 @@
+.modal-toggle:checked + .modal {
+    pointer-events: auto;
+    visibility: visible;
+    opacity: 1;
+}
+.modal {
+    pointer-events: none;
+    visibility: hidden;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    opacity: 0;
+    z-index: 999;
```

Comment:
I think this `z-index` is not needed, and is causing the `copy to clipboard` message to be always hidden.

```suggestion
```